docs/installing-windows.md

```diff
@@ -41,9 +41,12 @@ If you want to create a Windows installer from ChromeOS, see the [Installing Ven
     6. Right click LabConfig and create new DWORD (32 bit) called "BypassTPMCheck" and set it to 1
     7. Close regedit and command prompt and proceed with installation normally
 
-4. Install drivers:
+3. Install drivers:
    See [Coolstar's Windows Install Helper](https://coolstar.org/chromebook/windows-install.html) for drivers.
 
+4. (Optional) Install Chultrabook Tools for fan controls, backlight controls, System Information and more
```

Comment:
System Information should be all lowercase.

System Information -> system information